api/server/server.go

```diff
@@ -731,13 +731,20 @@ func New(ctx context.Context, opts ...Option) *Server {
 func WithPrometheus() Option {
 	return func(ctx context.Context, s *Server) error {
 		reg := promclient.NewRegistry()
-		reg.MustRegister(promclient.NewProcessCollector(os.Getpid(), "fn"),
+		reg.MustRegister(promclient.NewProcessCollector(promclient.ProcessCollectorOpts{
+			PidFn:     func() (int, error) { return os.Getpid(), nil },
```

Comment:
kinda nitty but our pid for our own process won't change so we don't need the func version for this one (I did not look if it went away), saves a syscall every collect interval or whatever (which is not much, so whatever)

Reply:
It went away :(